code/game/objects/structures/mirror.dm

```diff
@@ -374,7 +374,11 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/mirror/broken, 28)
 
 /obj/structure/mirror/magic/lesser/Initialize(mapload)
 	// Roundstart species don't have a flag, so it has to be set on Initialize.
-	selectable_races = get_selectable_species().Copy()
+	for(var/datum/species/species_type in subtypesof(/datum/species))
+		var/datum/species/species = species_type
+		if(species.check_roundstart_eligible())
```

Comment:
it feels kinda weird to have this be so dependent on config but i guess it ultimately makes sense